src/main/java/de/tum/in/www1/artemis/service/compass/utils/CompassConfiguration.java

```diff
@@ -25,10 +26,25 @@
 
     public final static double NO_MATCH_THRESHOLD = 0.1;
 
-    public final static double EQUALITY_THRESHOLD = 0.83;
+    // TODO CZ: decrease equality threshold again in the future
+    public final static double EQUALITY_THRESHOLD = 0.95;
 
     public final static double POORLY_ASSESSED_MODEL_THRESHOLD = 0.8;
 
     public final static double PARTIALLY_NAME_WEIGHT = 0.8;
 
+    // Confidence and coverage parameters
+    public static final double COVERAGE_THRESHOLD = 0.8;
+
+    public static final double CONFIDENCE_THRESHOLD = 0.75;
```

Comment:
I think it would be good to have a detailed comment for every of the constants.
E.g. `PARTIALLY_NAME_WEIGHT` is not clear to me.
I know not all constants were created by you, but it would help a lot for somebody who is not familiar with compass or its use in Artemis.

Reply:
You are right, I added a comment for every constant.